fixtures/apply_rules/output_report/report.json

```diff
@@ -1,4 +1,225 @@
 {
   "errors": [],
-  "warnings": []
+  "warnings": [
```

Comment:
this should not change. This is a fixture for tests that don't generate reporting

Reply:
I do agree and I didn't touch any of these files... or so I thought. I'm not sure why it happened. Let me rollback this change. Thanks for catching it.

Reply:
OKO, i just fixed it but I'm looking a bit more because I'm wondering if one of the bench is doing that. I used a simplified version of most of the IT tests for creating the bench but maybe I simplified too much for one.

Reply:
OK, I did some more changes to the bench (mostly the input folders for `apply-rules`). Should be better now.